src/libdredd/src/mutate_visitor.cc

```diff
@@ -37,6 +37,11 @@ MutateVisitor::MutateVisitor(const clang::CompilerInstance& compiler_instance)
     : compiler_instance_(compiler_instance),
       first_decl_in_source_file_(nullptr) {}
 
+bool MutateVisitor::IsTypeSupported(const clang::QualType qual_type) {
+  const auto* type = qual_type->getAs<clang::BuiltinType>();
```

Comment:
nit: I would call the variable `builtin_type`, to better distinguish it from `qual_type`.

Reply:
Done.